README.md

```diff
@@ -128,7 +128,7 @@ marp slide-deck.md -o converted.pdf
 
 > :information_source: All kind of conversions except HTML require [Google Chrome], [Chromium], [Microsoft Edge], or a Chromium based browser. When an unexpected problem has occurred while converting, please update your browser to the latest version or try installing [Google Chrome Canary].
 >
-> If you do not have these browsers, you can specify the path to a Chromium based browser using the `CHROME_PATH` environment variable. For example: `CHROME_PATH=$(which brave) marp --pdf slide-deck.md`
+> If you do not have [Google Chrome], [Chromium], or [Microsoft Edge] you can specify the path to a Chromium based browser using the `CHROME_PATH` environment variable. For example: `CHROME_PATH=$(which brave) marp --pdf slide-deck.md`
```

Comment:
Isn't it redundant to mention to the same browser set twice?

Reply:
You're right, it does look a bit redundant, but "these browsers" is actually referring to "Google Chrome, Chromuim, Microsoft Edge, or a Chromium based browser". Since it actually includes all Chromium based browsers, the rest of the sentence doesn't make sense

Reply:
Got it. But I feel repeat links (by wrapping with brackets) is unnecessary emphasis.

Reply:
Fixed